libmscore/barline.cpp

```diff
@@ -991,15 +1044,22 @@ qreal BarLine::layoutWidth(Score* score, BarLineType type)
                   break;
             case BarLineType::START_REPEAT:
             case BarLineType::END_REPEAT:
-                  w = score->styleP(Sid::endBarWidth) * .5
+                  w = (score->styleP(Sid::endBarWidth) * .5
                      + score->styleP(Sid::endBarDistance)
                      + score->styleP(Sid::repeatBarlineDotSeparation)
-                     + dotwidth * .5;
+                     + dotwidth * .5);
                   break;
             case BarLineType::END:
                   w = (score->styleP(Sid::endBarWidth) + score->styleP(Sid::barWidth)) * .5
                      + score->styleP(Sid::endBarDistance);
                   break;
+            case BarLineType::END_START_REPEAT:
```

Comment:
In principle we won't care about this code either because the END_START_REPEAT type won't ever appear in the score itself, and this code should not be needed in palette (?).  But it's harmless to keep around I guess.